app/main.py

```diff
@@ -17,3 +17,20 @@
 collection_of_coins = {1, 2, 25}
 
 # write your code here
+
+names_variables = [name for name in dir() if "__" not in name]
+print(f"In this module is - {len(names_variables)} some variables")
+
+mutable_type = [set, list, dict]
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+for var in names_variables:
+    if type(eval(var)) in mutable_type:
+        sorted_variables["mutable"].append(eval(var))
+    else:
+        sorted_variables["immutable"].append(eval(var))
```

Comment:
It's all good, but in this task, you are supposed to add variables to the dict manually, rewrite, please.